CMakeLists.txt

```diff
@@ -1616,6 +1623,26 @@ add_dependencies(mixxx-test mixxx-testdata)
 # library. Doing this would require initialization using Q_INIT_RESOURCE()
 # calls that are not present at the moment. Further information can be found
 # at: https://doc.qt.io/qt5/resources.html#using-resources-in-a-library
+option(DOWNLOAD_MANUAL_PDF "Download Manual PDF from Mixxx website" ON)
```

Comment:
Do we really want this on by default?

Reply:
I don't care. Before, the manual was included so I enabled it by default. The download is ~30MiB and we do pull other dependencies like libkeyfinder by default.

Reply:
I think it should be off by default. I don't care about downloading the manual PDF when building locally. This matters for packaging, not really for local builds.

Reply:
Ok, changed.